### PR TITLE
P#77246 Wrong use of translation function

### PR DESCRIPTION
**Pull request by dai-eastgate**
related to issue https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/306

chaged logs:

### CODE REVIEW

**Review by LongTrong-exe — APPROVED** [review]

**Review by LongTrong-exe — APPROVED** [review]

File: plugin/Field/FieldModuleCollectionDecoratorInternalAnnotations.php [hunk 1]
```diff
@@ -64,7 +64,7 @@ public function getFieldByModuleAndName(string $module, string $name): Field
 		$pField = parent::getFieldByModuleAndName($module, $name);
 		$newLabel = $this->getNewFields()[$module][$name] ?? null;
 		if ($newLabel !== null) {
-			$pField->setLabel(__($newLabel, 'onoffice-for-wp-websites'));
+			$pField->setLabel( sprintf( __( '%s', 'onoffice-for-wp-websites' ), $newLabel ) );
```
Comment (jmaas-onoffice) [inline plugin/Field/FieldModuleCollectionDecoratorInternalAnnotations.php:67]: How is this supposed to work? How should the WordPress translation know what `$newLabel` is to enable translation? :)

**Review by jmaas-onoffice — CHANGES_REQUESTED** [review]

File: plugin/Field/FieldModuleCollectionDecoratorInternalAnnotations.php (hunk 1 above)
Comment (dai-eastgate) [reply plugin/Field/FieldModuleCollectionDecoratorInternalAnnotations.php:67]: it has been translated, for example line 49 just $newLabel is enough, it needs to be translated before using it.
Can you please test it? I tested it and it still translates normally :)
As for the words that have not been translated before, I will check them carefully in this PR

Comment (jmaas-onoffice) [reply plugin/Field/FieldModuleCollectionDecoratorInternalAnnotations.php:67]: If `$newLabel` is already translated, why are we using `__( '%s', 'onoffice-for-wp-websites' )` here? :)

**Review by jmaas-onoffice — APPROVED** [review]

**Comment by dai-eastgate** [comment]
@jmaas-onoffice Sorry, I forgot to submit a review request here

**Pull request merged**